x/concentrated-liquidity/incentives.go

```diff
@@ -742,6 +764,20 @@ func moveRewardsToNewPositionAndDeleteOldAcc(ctx sdk.Context, accum accum.Accumu
 		return err
 	}
 
+	ctx.EventManager().EmitEvents(sdk.Events{
```

Comment:
Note: I added these events to assist with debugging https://github.com/osmosis-labs/osmosis/issues/5087